### DISCUSSION TITLE
I like the changes made in the alpha

### DISCUSSION THREAD

**Discussion by josephtoles** [hylang/hy#2180] [General]
I have been using Hy's stable release for the past couple years. I just switched to the alpha. There are a lot of things I like about it.
- I like that the redundant brackets on `import` and `require` statements have been removed.
- I like that `&rest` has been replaced by `#*`. It's shorter, prettier and more Pythonic.
- I reluctantly accept that limiting `if` to three arguments is the right way to go, from a language design standpoint. The previous behavior of `if` confused new people when I showed them Hy.
- I look forward to using optional arguments in `defmacro!`. There have been times in the past that I wish it had that feature.
- The shortening of `HySymbol` to `Symbol` (and similar name changes) makes things shorter and prettier.
- I was surprised how `-` is distinguished from `_` in places where it previously wasn't. I thought that would never happen. This appears to be a very good change for the language.

**Comment by allison-casey** [comment]
Glad you're liking the changes! Let us know if you have any other ideas to improve the language. We may be getting close to release, but there's always room to improve.
